CHANGELOG.md

```diff
@@ -3,6 +3,77 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[2.0.0](https://github.com/nf-core/pixelator/releases/tag/2.0.0)] - 2024-05-27
+
+This release is a major update of the nf-core/pixelator pipeline. It brings in support for the
+Proximity Network Analysis (PNA) workflow. Any run of the pipeline will now run the appropriate
```

Comment:
"It brings in support for the
Proximity Network Analysis (PNA) workflow" --> "It brings in support for the
Proximity Network Analysis (PNA) workflow in addition to the Molecular Pixelation (MPX) workflow." (just for clarity)